### PR TITLE
Edited contributing.md

### PR DESCRIPTION
**Pull request by dbkonxepts**
Converted the **categories** section of the contribution to a _step_ i.e 4th step, so it has more emphasis

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -40,9 +40,10 @@ company_name
 }
 ```
 
-## Categories
-Here are the current categories we recognize. If your company category is not reflected here, feel free to use a new category. If you do so, kindly edit this file in your PR as well.
 
+4. If your company category is not reflected in categories section *below*, feel free to use a new category. If you do so, kindly edit this file (**/contributing.md**) in your PR as well.
+
```
Comment (onyekvchi) [inline contributing.md]: heyy @dbkonxepts, great work here! Can you please link to `contributing.md` directly here?

File: contributing.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ company_name
 |       company_name.svg
 ```
 
-3. Finally, in **js/logos.json**, add a `JSON` entry for your company
+3. In **js/logos.json**, add a `JSON` entry for your company
```
Comment (onyekvchi) [inline contributing.md]: Please also link to `logos.json` here

File: contributing.md (hunk 1 above)
Comment (dbkonxepts) [reply contributing.md]: Alright, coming through.

Comment (dbkonxepts) [reply contributing.md]: Sorry, can you rephrase?
_Link to the same file?_

File: contributing.md (hunk 2 above)
Comment (dbkonxepts) [reply contributing.md]: Alright.

File: contributing.md (hunk 1 above)
Comment (onyekvchi) [reply contributing.md]: Ah, seen. No need to link to this
